RELEASE.md

```diff
@@ -0,0 +1,44 @@
+# Team
+
+The release team is made up of the following:
+
+* Rotating release engineer from one of the development teams
```

Comment:
One concern that has come up is a natural asymmetry inside the teams. In particular, the ledger team often makes changes that require integrating in ouroboros-network and node, and hence ledger might be in the best position to do the integration. We (ledger) do keep a changelog, but it is still probably easier for us. If we did use a different procedure, where the team who "caused" the work did the integration, then I would love it if we could also nominate folks on the other teams (for the same duration) for pairing up with the release engineer when they need help. If we name folks explicitly, it makes it easier to ask for help and it makes it easier to justify the time providing help.